notebooks/07_emg_hand_gesture_classification.ipynb

```diff
@@ -0,0 +1,317 @@
+{
```

Comment:
<p>NITs: </p><p> </p><p>Process these signals into covariance matrices (plural) ?</p><p> </p><p>How do the different methods (does -&gt; do)</p><p> </p><p>tangeant -&gt; tangent</p><p>euclidean -&gt; Euclidean</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/899/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>